featuretools/primitives/utils.py

```diff
@@ -39,14 +39,14 @@ def get_transform_primitives():
 
 def list_primitives():
     trans_names, trans_primitives = _get_names_primitives(get_transform_primitives)
-    trans_dask = [primitive.dask_compatible for primitive in trans_primitives]
+    trans_dask = ['dask' in primitive.compatibility for primitive in trans_primitives]
     transform_df = pd.DataFrame({'name': trans_names,
                                  'description': _get_descriptions(trans_primitives),
                                  'dask_compatible': trans_dask})
     transform_df['type'] = 'transform'
 
     agg_names, agg_primitives = _get_names_primitives(get_aggregation_primitives)
-    agg_dask = [primitive.dask_compatible for primitive in agg_primitives]
+    agg_dask = ['dask' in primitive.compatibility for primitive in agg_primitives]
```

Comment:
add koalas?
`trans_koalas = ['koalas' in primitive.compatibility for primitive in trans_primitives]`